spec/domain/operations/benefit_sponsors/employer_profile/bulk_ce_upload_spec.rb

```diff
@@ -6,7 +6,7 @@
 
   describe '#call' do
     let(:bucket_name) { 'ce-roster-upload' }
-    let(:uri) { "urn:openhbx:terms:v1:file_storage:s3:bucket:#{bucket_name}{#sample-key}" }
+    let(:s3_uri) { "f55679fe-34ca-426c-b68b-d5d92f16255c" }
```

Comment:
is this only for test? it will not affect the running code ?

Reply:
yes this is only for test, we pass in new arg `s3_uri`.